commands/exec.go

```diff
@@ -54,11 +53,6 @@ func (e *KoolExec) detectTTY() {
 	if !isTerminal {
 		e.composeExec.AppendArgs("-T")
 	}
-
-	if aware, ok := e.composeExec.(compose.TtyAware); ok {
```

Comment:
`docker compose` has better tty support / awareness?

Reply:
No, it's the same as it always was. What we removed is the double-trouble DinD option, where we had to set it twice.